delivery_correos_express/tests/test_delivery_correos_express.py

```diff
@@ -28,7 +28,7 @@ def setUpClass(cls):
             }
         )
         cls.product = cls.env["product.product"].create(
-            {"type": "product", "name": "Test product"}
+            {"is_storable": True, "name": "Test product"}
```

Comment:
Add the flag, but don't remove the type.

Reply:
The product type does not exist in the 18

Reply:
```suggestion
            {"type": "consu", "is_storable": True, "name": "Test product"}
```
Is true that now `product` type does not exist but it replace by `consu` + `is_storable`. Anyways it is true that type is set to `consu` by default but better to set it just to clarify.